### PR TITLE
Spelling and added surnames for citation purposes

### PR DESCRIPTION
**Pull request by Lextuga007**
## Description
Spelling correction for epistemological.

I've also added your surnames @NatalieZelenka and @ninadicara as I wanted to cite you in this work 😄 

## Checklist (contributor)

- [x] Marked as draft
- [X] Linked to issue (#142 and #143)
- [X] Filled in description
- [X] Gave this Pull Request a descriptive title
 
## Checklist (maintainer)

- [ ] Built site locally.
- [ ] Changes are as discussed in issue.
- [ ] New pages are linked in the TOC/navigation?
- [ ] Have we added the contributor to our all-contributors?
- [ ] One sentence per line

### CODE REVIEW

**Comment by ninadicara** [comment]
Zoë you're the best! Thank you - will take a look tomorrow ❤️

**Comment by ninadicara** [comment]
PS @Lextuga007 - I noticed this PR is still in draft - shall I wait until you've finished or were you hoping to merge it now? N

**Comment by Lextuga007** [comment]
I think this is done now, thanks 😃

**Review by ninadicara — APPROVED** [review]

**Pull request merged**
